src/routes/$repo.$.tsx

```diff
@@ -201,7 +202,7 @@ function UpdateNotifier() {
 }
 
 export default function Repo() {
-  const data = useLoaderData<RepoData>()
+  const data = useLoaderData<typeof loader>()
```

Comment:
How is this different?

Reply:
It uses baked in remix type inference instead of an explicit type. I had some issues with the typings, but changing this solved the issues.

Reply:
Oh nvm, it didn't solve the issue

Reply:
I solved it by using this package: https://www.npmjs.com/package/remix-typedjson